airbyte-integrations/connectors/source-mongodb-v2/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.2.4'
+    cdkVersionRequired = '0.4.0'
```

Comment:
@stephane-airbyte has a [PR](https://github.com/airbytehq/airbyte/pull/32125/files#diff-b2ee0596a72284c57a3691e096fe6c05e004dbb8b0dba45b3ef19beaadbb7c53) with this also .